docs/dpdk-configuration.md

```diff
@@ -0,0 +1,77 @@
+<!--
+SPDX-License-Identifier: Apache-2.0
+Copyright 2022 Intel Corporation
+-->
+
```

Comment:
lot of users faced issue as they missed on prerequisite requirements for enabling virtualization, without that this dpdk script/tools wont produce output given below. pls mention those as well

Reply:
Good point. I am going to add that